python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -1031,55 +1017,23 @@ def _getIOTimeOut(self):
             _timeOut = 5. # 5sec hardcoded default
         return _timeOut
 
-    def _getWriteAccess(self):
-        """ tries to get write access to the object.
-        Raise LockingError (or so) on fail """
-        root = self._getRoot()
-        reg = root._getRegistry()
+    def _getSessionLock(self):
```

Comment:
Why is this used? We currently get a lock on loading the object or are there plans to change this?